models/selectArticles.models.js

```diff
@@ -30,16 +30,39 @@ exports.selectArticles = (sort_by = "created_at", order = "desc") => {
 };
 
 exports.patchArticleVotesById = (article_id, inc_votes) => {
-  return db.query(
-    `UPDATE articles 
+  return db
+    .query(
+      `UPDATE articles 
      SET votes = votes + $1
      WHERE article_id = $2
      RETURNING *;`,
-    [inc_votes, article_id]
-  ).then((data) => {
-    if(data.rows.length === 0){
-      return Promise.reject({status: 404, msg: "Article Id does not exist"})
-    }
-    return data.rows[0];
-  });
-};
\ No newline at end of file
+      [inc_votes, article_id]
+    )
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "Article Id does not exist",
+        });
+      }
+      return data.rows[0];
+    });
+};
+
+exports.createComment = (article_id, newComment) => {
+  const { username, body } = newComment;
+  return db
+    .query(
+      `INSERT INTO users (username, name, avatar_url) VALUES ($1, $2, $3)`,
+      [username, username, "placeholder"]
+    )
```

Comment:
This should not be happening for this endpoint. Only valid users should be able to vote so if the user does not exist we should be throwing an error.